data_collection/web/guides.rst

```diff
@@ -221,187 +221,184 @@ indicate no value.
 
 
 
+E-commerce v2
```

Comment:
I think we should call it just `E-commerce` without the `v2` mark.
In the guides, we want to inform customers about our suggested approach to implementing JSTC methods. We don't want to mention the old, deprecated way of tracking e-commerce here.